cirq-pasqal/cirq_pasqal/json_test_data/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Cirq Developers
+# Copyright 2021 The Cirq Developers
```

Comment:
nit: I am not sure if it matters, but this __init__ file renaming is weird.  It is taking the top-level __init__ to the json_test_data __init__ which makes the diff look funny and screws up git blame.  I am not sure it matters too much though, if we can't figure out how to fix it.